infrastructure/transform/main.tf

```diff
@@ -65,6 +65,45 @@ resource "azurerm_data_factory" "adf" {
   }
 }
 
+
+resource "azurerm_data_factory_pipeline" "pipeline" {
+  name            = "databricks-pipeline-${var.naming_suffix}"
+  data_factory_id = azurerm_data_factory.adf.id
+  activities_json = <<JSON
+[
+  {
+    "name": "DatabricksPythonActivity",
+    "type": "DatabricksSparkPython",
+    "typeProperties": {
+        "pythonFile": "dbfs:/entrypoint.py",
+        "libraries": [
+            {
+                "whl": "dbfs:/src-0.0.1-py3-none-any.whl"
+            }
+        ]
+    },
+    "linkedServiceName": {
+        "referenceName": "ADBLinkedServiceViaMSI",
+        "type": "LinkedServiceReference"
+    }
+  }
+]
+  JSON
+}
+
+resource "databricks_dbfs_file" "dbfs_file_upload" {
```

Comment:
Don't we need to specify "overwrite" flag?

Reply:
We don't need overwrite flag as it does that by default. 
Every time we deploy the file will be deployed to DBFS regardless of it has changed or not changed.